internal/views/authors/index.html

```diff
@@ -18,20 +18,46 @@ <h3>Autores Cadastrados</h3>
         <tbody>
         {{range .Authors}}
             <tr>
-                <td>{{.ID}}</td>
+                <td class="idAuthor">{{.ID}}</td>
                 <td>{{.Name}}</td>
                 <td>
                     <a href="/authors/{{.ID}}/edit">Editar</a>
+                    <button class="btnRm" data-id="{{.ID}}">Remover</button>
```

Comment:
```suggestion
                    <button class="btn-rm" data-id="{{.ID}}">Remover</button>
```

Vai depender um pouco de cada projeto, mas em geral classes são escritas [usando a notação `kebab-case`](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Code_style_guide/HTML#class_and_id_names).